### DISCUSSION TITLE
Adding new 3Dconnexion devices

### DISCUSSION THREAD

**Discussion by JakubAndrysek** [JakubAndrysek/PySpaceMouse#26] [General]
If you want to add an additional device.
- fork the repo
- find out the VID + PID
- try to map buttons and axis to the code
	- 	this is Windows helper, code: [raw_data.py](https://github.com/rene-aguirre/pywinusb/blob/master/examples/raw_data.py)
- example of configured device: [SpaceMouse USB](https://github.com/JakubAndrysek/PySpaceMouse/blob/9941f2ec21a09b4d4ad72b373f0059d4270b2e56/pyspacemouse/pyspacemouse.py#L469-L487)
- create PR

Try to look into the parent's repo (forked from) - [more details](https://github.com/johnhw/pyspacenavigator/issues/1#issuecomment-2093970390) - Windows only (have to be modified)

**Comment by peter-mitrano-ar** [comment]
Updated docs on adding a new device are found here: https://spacemouse.kubaandrysek.cz/CONTRIBUTING/
